src/utilities/calculator.js

```diff
@@ -162,14 +162,14 @@ const decliningRepaymentCalculatorForThreeMonths = (
   const residual = productPrice - actualDownpayment
   const normalInstallment = residual / count
   const normalInstallmentAndInterest =
-    normalInstallment + (interest / 100) * residual
+    normalInstallment + (params.interest / 100) * residual
```

Comment:
Why are we having calc changes here?

Reply:
It was Taofeek that said I should

Reply:
It should be inside this PR, should be in your other PR

Reply:
should I remove it or just leave it